integration/spark/app/integrations/container/applicationLevelCompleteJob.json

```diff
@@ -0,0 +1,16 @@
+{
+  "eventType": "COMPLETE",
+  "job": {
+    "namespace": "generic-namespace"
+  },
+  "run": {
+    "facets": {
+      "parent": {
+        "job": {
```

Comment:
Shouldn't complete job contain runUid in parent of complete job?
Is it possible for a test to have same run id in different events or verify afterwards with get events emitted that run ids match?  It would be nice to test that events can be linked at the backend then.

Reply:
>Shouldn't complete job contain runUid in parent of complete job?

Not sure what you mean - any event type should contain runId of the parent job. I don't include it in current comparison of events, because it's a random UUID and we aren't able to use Jinja to preprocess the events. It will change when mockserver is removed.